e2e/test/preview.test.ts

```diff
@@ -13,6 +13,22 @@ test('user can see single preview tab', async ({ page }) => {
   await expect(preview.getByText('Index page')).toBeVisible();
 });
 
+test('user can reload the page', async ({ page }) => {
```

Comment:
`page` doesn't sound right here. Maybe something like:

```suggestion
test('user can reload a preview tab', async ({ page }) => {
```